superdesk/users/services.py

```diff
@@ -229,8 +229,15 @@ def get_avatar_renditions(self, doc):
         renditions = get_resource_service("upload").find_one(req=None, _id=doc)
         return renditions.get("renditions") if renditions is not None else None
 
+    def is_email_required(self, doc):
```

Comment:
this would be better handled with custom validation, like we have for `username_pattern`, so it's also checked for updates